rust/fastsim-core/src/vehicle.rs

```diff
@@ -49,45 +51,26 @@ lazy_static! {
         self.mc_peak_eff()
     }
 
-    // TODO: refactor this to have a non-py and `_py` version
```

Comment:
@Romia87 , I made some edits to your changes and the code is correct now.  I had actually left that `TODO` note for myself months (maybe even years ???) ago and forgotten about it so it's good you got this to happen!